### PR TITLE
added models related to listings

### PR DESCRIPTION
**Pull request by caraencabo**
## Changes Made

Added models and migrations for:

- Listing, Accommodation, Experience, Calendar, Media, Review

Added enums for constants:

- AccommodationType, Amenity, ExperienceType, Inclusion, ListingStatus

## Purpose

To add models and migrations necessary for other transactions.

## Related Task/Issue

[Ticket Link](https://framgiaph.backlog.com/view/SBNB-57)

## Screenshots

N/A

## New Dependencies (if applicable)

N/A

## Additional Information (if applicable)

N/A

### CODE REVIEW

**Review by faith-rosalijos — APPROVED** [review]
Looks good to me.

File: backend/database/migrations/2024_03_05_014548_create_reviews_table.php [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration {
+    /**
+     * Run the migrations.
+     */
+    public function up(): void {
+        Schema::create('reviews', function (Blueprint $table) {
+            $table->id();
+            // $table->foreignId('user_id')->constrained();
```
Comment (brettjosef) [inline backend/database/migrations/2024_03_05_014548_create_reviews_table.php:14]: should this be commented?

**Review by brettjosef — COMMENTED** [review]
User id commented

**Review by brettjosef — APPROVED** [review]
This looks good

**Pull request merged**